README.md

```diff
@@ -7,10 +7,13 @@ Since Touch Bar is just a flat panel, there is no feedback other than a visual c
 
 By adding a haptic feedback, which is vibrating a trackpad not Touch Bar itself, it helps to improve the usage of Touch Bar. It is also optionally playing a sound effect or flashing the screen like a visual bell on a terminal.
 
-Usage
------
+Installation
+----
 
-Download the latest pre-built application binary from [Releases](https://github.com/niw/HapticKey/releases) page. Note that these pre-built application binaries are not signed so you need to allow to execute it on Security & Privacy settings pane in System Preferences.
+- Via Homebrew: `brew cask install haptickey`
```

Comment:
Thank you for updating this section! One minor comment here about Homebrew Cask.

I personally didn’t manage Homebrew Cask and the one currently available is contributed by @brianmorton as like homebrew/homebrew-cask#53330 as of #8.
I don’t think Cask is automatically updated so I want to mention about this fact as like
```
**NOTE:** Homebrew Cask may not be updated and install the older version.
However it will automatically update to the latest one when you restart the application.
```
I remember Homebrew Cask had a plan to support [Sparkle](https://sparkle-project.org/), which is a very common software updater that HaptickKey is using to distribute the new release, but it may not be available yet or I don't know if Homebrew Cask project still try to support it or not...

Reply:
Cask updates are manual affairs currently, so if there is a new version of HapticKey available and the binaries do not autoupdate themselves, somebody would need to submit a PR to homebrew cask in order for the version installed via homebrew to be updated/current.

Reply:
@niw Do they already support `appcast`s and Sparkle since 2016? https://github.com/Homebrew/homebrew-cask/issues/16689

`appcast`s and Sparkle are mentioned in the latest version of the documentation, at least:

https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md

Reply:
Yes they are supported, but it is still a manual effort for updates (somebody has to submit a PR).

Reply:
Yeah, HapticKey has the app cast URL specified in [`Info.plist`](https://github.com/niw/HapticKey/blob/master/HapticKey/Info.plist)

However, as I see Homebrew Cask implementation and also as @brianmorton mentioned, seems like for now, it’s a supplemental information and not used for updating each package.

Reply:
Hm, too bad there's not a bot / other GitHub automation for this already. (Maybe GitHub Actions could be used here?)

Reply:
A couple of us from homebrew are working on automation for cask updates, but it is difficult as there are 4000+ casks and every developer releases their stuff differently ;) Any help is welcomed, however.

Reply:
Oh cool, didn't find that project with a few searches of the issues. Is there a public link for this effort?

Reply:
@karlhorky https://github.com/brianmorton/cask-automation to scan for outdated casks, `cask-repair` from https://github.com/vitorgalvao/tiny-scripts to submit PRs, and this thread https://github.com/Homebrew/homebrew-cask/issues/41240

Reply:
@karlhorky also this https://github.com/Homebrew/homebrew-cask/issues/52837